troubadix/plugins/script_version_and_last_modification_tags.py

```diff
@@ -129,6 +129,41 @@ def check_content(
                 file=nasl_file,
                 plugin=self.name,
             )
+        else:
+            date_str = match_last_modified.group("value")
+            format_left = "%Y-%m-%d %H:%M:%S %z "
+            format_right = "(%a, %d %b %Y)"
+            try:
+                date_left = datetime.datetime.strptime(
+                    date_str[:26], format_left
+                )
+                date_right = datetime.datetime.strptime(
+                    date_str[26:], format_right
+                )
+                week_day_parsed = date_right.strftime("%a")
+            except ValueError:
+                yield LinterError(
+                    "False or incorrectly formatted modification_date.",
+                    file=nasl_file,
+                    plugin=self.name,
+                )
+                return
+            week_day_str = date_str[27:30]
+            # Wed, 29 Nov 2017
+            if date_left.date() != date_right.date():
+                yield LinterError(
+                    "The modification_date consists of two different dates.",
+                    file=nasl_file,
+                    plugin=self.name,
+                )
+            # Check correct weekday
+            elif week_day_str != week_day_parsed:
+                yield LinterError(
```

Comment:
Maybe `return` here too instead of `elif`? But that's just a personal preference :)

Reply:
Done in 9d6c32dba64917df1e278f176bcf5ed0d41e8af6